bootstrap.py

```diff
@@ -247,6 +247,16 @@ def get_puppet_version():
             return puppet_version
 
 
+def is_fips():
+    """
+    Checks to see if the system is FIPS enabled.
+    """
+    fips_file = open("/proc/sys/crypto/fips_enabled", "r")
```

Comment:
I wonder if we should handle systems where this file does not exist. (like a CentOS container running on Debian).

Reply:
Why wouldn't it exist?  (though if we are going to pretend bootstrap works on non-RHEL, we'll need a whole nother 5 pull requests)

Reply:
I think an EL/Fedora container running on non-EL/Fedora is the only way this could happen.

Reply:
but the FIPS attribute is set at the kernel level. I suspect `/proc/sys/crypto/fips_enabled` exists on a Debian system. (Though I don't have one in front of me to confirm)

Reply:
Yeah, it's dependent on the docker host. Every single OS image (centos, rhel, debian, alpine) has that file when I run docker on Fedora, but none of the images has it when I run on non-EL/Fedora, such as on OS X, for instance. To clarify, running a RHEL container does not even have that file if the container is running on an OS that doesn't have that file.

Reply:
Yeah, exactly. It's a kernel thing, so a container running on a non-EL/Fedora kernel will not have this.

It's not like I like registering containers anyways...